### DISCUSSION TITLE
[F] Column page breaks

### DISCUSSION THREAD

**Discussion by StefanNedelchev** [exceljs/exceljs#2453] [Ideas & Proposals]
## 🚀 Feature Proposal

Column page breaks (similar to row breaks but these are applied on columns instead of rows)

## Motivation

The XLSX format supports column breaks just like row breaks but this library only implements only the latter.

## Example

```ts
// add row break after the row (currently supported)
sheet.getRow(9).addPageBreak();
// add column break after the column (currently not supported)
sheet.getColumn('C').addPageBreak();
```

![image](https://github.com/exceljs/exceljs/assets/15238282/6671722f-c0e5-4953-b9b3-961c1e171be0)

**Comment by StefanNedelchev** [comment]
Also a question - do you accept PRs if I implement this feature?

**Comment by zurmokeeper** [comment]
@StefanNedelchev hello, I'm interested in this feature, you can take a look at the project([@zurmokeeper/exceljs](https://github.com/zurmokeeper/excelize)) and if you can implement it, I can merge it here if it's useful to very many people. Thank you!

**Reply by zzx58-027** [reply]
很需要这个功能, 请问什么时候会将此功能合并?
Really need this function. Please, when will this function be merged?

**Reply by zzx58-027** [reply]
我看到您 fork exceljs 的仓库, 感谢您的维护!!
I saw your fork exceljs warehouse and thank you for your maintenance!!

**Reply by zurmokeeper** [reply]
> 很需要这个功能, 请问什么时候会将此功能合并? Really need this function. Please, when will this function be merged?

很久之前就已经合并发布了，你可以试试。
It was merged and released a long time ago, so you can try it.

**Comment by StefanNedelchev** [comment]
> @StefanNedelchev hello, I'm interested in this feature, you can take a look at the project([@zurmokeeper/exceljs](https://github.com/zurmokeeper/excelize)) and if you can implement it, I can merge it here if it's useful to very many people. Thank you!

https://github.com/zurmokeeper/excelize/pull/13
